docs/current/guides/235290-troubleshooting.md

```diff
@@ -0,0 +1,38 @@
+---
+slug: /235290/troubleshooting
+displayed_sidebar: 'current'
+category: "guides"
+tags: []
+authors: ["Vikram Vaswani"]
+date: "2023-04-27"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Troubleshooting Dagger
+
+This page describes problems you may encounter when using Dagger, and their solutions.
+
+## Dagger pipeline is unresponsive with a BuildKit error
+
+A Dagger pipeline may hang or become unresponsive, eventually generating a BuildKit error such as `buildkit failed to respond` or `container state improper`.
+
+To resolve this error, you must stop and remove the Dagger Engine container and clear unused volumes and the cache:
+
+1. Stop and remove the Dagger Engine container:
+
+  ```shell
+  DAGGER_CONTAINER_NETWORK_NAME=`docker ps --filter "name=^dagger-engine-*" --format '{{.Names}}'`
+  docker stop $DAGGER_CONTAINER_NETWORK_NAME
```

Comment:
Could we make this docker-agnostic or is it implied that the audience will know to substitute `docker` for `podman` if required?

Reply:
Good question. WDYT @gerhard @kpenfound ?

Reply:
By default, Dagger relies on Docker for provisioning the Engine, so using `docker` in this context seems right to me. Resolving this. Please re-open if you disagree @moljac024.

I see that we have added a note about custom container runtimes. To me, this is more confusing than helpful. Will address that separately.